### PR TITLE
Make super slow tests integration tests

### PR DESCRIPTION
**Pull request by IAlibay**
Adds `--integration` or `OFE_INTEGRATION_TESTS` as triggers for the super slow tests.

### CODE REVIEW

**Comment by IAlibay** [comment]
Do we want to make these run on CLI? It wasn't clear that folks _should_ ever do that.

File: openfe/tests/conftest.py [hunk 1]
```diff
@@ -12,30 +12,62 @@
...
-           "`OFE_SLOW_TESTS` set to `True` to run")
-    skip_slow = pytest.mark.skip(reason=msg)
-    for item in items:
-        if "slow" in item.keywords:
-            item.add_marker(skip_slow)
+    config.pluginmanager.register(SlowTest(config), "slow")
+    config.addinivalue_line("markers", "slow: mark test as slow")
+    config.pluginmanager.register(IntegrationTest(config), "integration")
+    config.addinivalue_line(
+            "markers", "integration: mark test as long integration test")
```
Comment (richardjgowers) [inline openfe/tests/conftest.py:70]: shouldn't `--integration` also run the slows tests?

File: openfe/tests/conftest.py [hunk 2]
```diff
@@ -12,30 +12,62 @@
...
+        msg = ("need --runslow pytest cli option or the environment variable "
+           "`OFE_SLOW_TESTS` set to `True` to run")
+        skip_slow = pytest.mark.skip(reason=msg)
+        for item in items:
+            if "slow" in item.keywords:
+                item.add_marker(skip_slow)
+
+
+class IntegrationTest:
+    """Plugin for a fixture that skips very slow integration tests"""
```
Comment (richardjgowers) [inline openfe/tests/conftest.py:35]: these docs strings don't tell me how to use this.  something like `for tests marked with @pytest.mark.integration`

**Review by richardjgowers — CHANGES_REQUESTED** [review]

File: openfe/tests/conftest.py (hunk 1 above)
Comment (IAlibay) [reply openfe/tests/conftest.py]: Fixed

File: openfe/tests/conftest.py (hunk 2 above)
Comment (IAlibay) [reply openfe/tests/conftest.py]: Much docstring was added 🙃

**Comment by IAlibay** [comment]
oops forgot to update CI

**Pull request merged**